kodilla-exeption/src/main/java/com/kodilla/exception/test/RouteFinder.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.exception.test;
+
+import com.kodilla.exception.io.FileReaderException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class RouteFinder {
+
+    public void findRoutes(Flight flight, Map<String, Boolean> routes) throws RouteNotFoundException {
```

Comment:
Map<String, Boolean> routes - to powinno byc polem private final w tej klasie.

Reply:
pytamy sie, czy mozna na dane lotnisko doleciec - skomplikowales. Wylatujemy z aktualnego lotniska.